src/calendar/grid/index.tsx

```diff
@@ -111,7 +120,8 @@ export default function Grid({
               scope="col"
               className={clsx(styles['calendar-grid-cell'], styles['calendar-day-header'])}
             >
-              {renderDayName(locale, dayIndex)}
+              <span aria-hidden="true">{renderDayName(locale, dayIndex, 'short')}</span>
+              <span className={styles['visually-hidden']}>{renderDayName(locale, dayIndex, 'long')}</span>
```

Comment:
How about changing "visually-hidden" to "screenreader-only"? I think it makes the code better readable by answering the question "why?" instead of "what?".

Reply:
We have "visually-hidden" in Formfield too. Let's refactor it as a whole as followup of this ticket.